theme/tags.css

```
@@ -0,0 +1,29 @@
+.tag-os {
+    font-size: 11px;
+    padding: 1px 8px;
+    border-radius: 10px;
+    border: 1px solid;
+    color: #ffffff !important;
```

Stylelint (17.3.0)

[error] Unexpected duplicate "color" (declaration-block-no-duplicate-properties)

(declaration-block-no-duplicate-properties)